simulation/src/test/java/com/palantir/dialogue/core/SimulationTest.java

```diff
@@ -103,6 +104,11 @@
     @ParameterizedTest
     @interface SimulationCase {}
 
+    @BeforeAll
+    static void beforeAll() {
+        Tracer.setSampler(() -> false);
+    }
```

Comment:
I had this in @Before? https://github.com/palantir/dialogue/blob/707b92aaeac49ff316b770837770a2edca07504d/simulation/src/test/java/com/palantir/dialogue/core/SimulationTest.java#L624

Reply:
doh, missed that.

Reply:
Ressurrected this from an old branch, I think that may not have existed when I wrote this, also very possible I missed it.